apps/cms/src/admin/views/MerchOverview.tsx

```diff
@@ -19,6 +59,29 @@ const MerchOverview: AdminView = ({ user, canAccessAdmin }) => {
       <Button el="link" to={"/admin"} buttonStyle="primary">
         Go to Main Admin View
       </Button>
+      <p style={{ paddingTop: 20 }}>{`Current state of merch store: ${
+        loading ? "..." : isStoreDisabled ? "Disabled" : "Live"
+      }`}</p>
+      {SHOW_DISPLAY_TEXT_INPUT && (
+        <textarea
+          value={displayText}
+          onChange={(e: ChangeEvent<HTMLTextAreaElement>) =>
+            setDisplayText(e.target.value)
+          }
+          placeholder="Enter display text"
+          rows={4}
+          cols={50}
+        />
+      )}
```

Comment:
@Iyzyman CharkraUI is not installed for payload, how do we proceed with the styling